### PR TITLE
Documentation, Chart, and Manifest updates for v1.13.0 release

### PR DESCRIPTION
**Pull request by jdn5126**
**What type of PR is this?**
release updates

**Which issue does this PR fix**:
N/A

**What does this PR do / Why do we need it**:
This PR mirrors https://github.com/aws/amazon-vpc-cni-k8s/pull/2394 in updating the relevant docs, charts, and manifests for the v1.13.0 release.

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:
N/A

**Testing done on this change**:
N/A

**Automation added to e2e**:
N/A

**Will this PR introduce any new dependencies?**:
N/A

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:
N/A

**Does this change require updates to the CNI daemonset config files to work?**:
N/A

**Does this PR introduce any user-facing change?**:
N/A

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -669,6 +657,20 @@ Default: `false`
...
+
+Type: Boolean as a String
+
+Default: `false`
+
+Specifies whether PODs in an IPv4 cluster support IPv6 egress. If env is set to `true`, range `fd00::ac:00/118` is reserved for IPv6 egress.
+
+This environment variable must be set for both the `aws-vpc-cni-init` and `aws-node` containers in order for this feature to work properly. This feature also requires that the node has an IPv6 address assigned to its primary ENI, as this address is used for SNAT to IPv6 endpoints outside of the cluster. If the configuration prerequisites are not met, the `egress-cni` plugin is not enabled and an error log is printed in the `aws-node` container.
+
+Note that enabling/disabling this feature only affects whether newly created pods have an IPv6 interface created. Therefore, it is recommended that you reload existing nodes after enabling/disabling this feature. Also note that if you are using this feature in conjunction with `ENABLE_POD_ENI` (Security Groups for Pods), the security group rules will NOT be applied to egressing IPv6 traffic.
```
Comment (haouc) [inline README.md:672]: Can you elaborate the work `reload`? Does this mean `terminate`?

**Review by orsenthil — DISMISSED** [review]
LGTM.

If you use 'reboot' instead of 'reload', I am with +1 with it.

File: README.md (hunk 1 above)
Comment (jdn5126) [reply README.md:672]: Yeah, it means "reboot". I looked at our official docs and it looks like they use "reboot", so I will update this.

Comment (jdn5126) [reply README.md]: Updated in latest revision

**Review by orsenthil — APPROVED** [review]
LGTM

**Pull request merged**